Lib/sqlite3/dbapi2.py

```diff
@@ -55,16 +55,24 @@ def TimestampFromTicks(ticks):
 collections.abc.Sequence.register(Row)
 
 def register_adapters_and_converters():
+    from warnings import warn
+
+    msg = "The default {what} are deprecated as of Python 3.12"
```

Comment:
```suggestion
    msg = ("The default {what} are deprecated as of Python 3.12; "
           "see the sqlite3 documentation for suggested replacement recipes")
```

Might we want to briefly mention the replacement here to give users a starting point for what to do about this?

Reply:
+1